connectors/s3-connector/src/test/scala/zio/connect/s3/LiveS3ConnectorSpec.scala

```diff
@@ -33,6 +33,17 @@ object LiveS3ConnectorSpec extends S3ConnectorSpec {
       })(ls => ZIO.attempt(ls.stop()).orDie)
     )
 
+  lazy val localStackContainerUsWest: ZLayer[Scope, Throwable, LocalStackContainer] =
```

Comment:
I don't think we need another container. We can use the existing one and create another s3 that has a different region which will use the same localstack container.